camunda-formio-tasklist-vue/docs/StyleGuide.md

```diff
@@ -1,8 +1,16 @@
-## Style Guide
+# Style Guide
 
-1. When writing css, always start with a prefix like `cft-` to avoid conflicts with other libraries
-when integrating.
+1. When writing css, always start with a prefix like `cft-` to avoid conflicts
+with CSS in other projects when integrating vue component.
 2. Import the components in relative order.
 3. When naming new Components follow Pascal-case.
 4. Follow the [VueJS Style Guide when developing components](https://vuejs.org/v2/style-guide/).
 5. On developing your code, follow the [Eslint rules specified](../.eslintrc.js)
+
+## Guidelines for raising PRs
+
+1. There are two protected branches: master and dev. dev is for continuous development
+and master branch as stable branch for the project which contains released version code.
+2. For any new changes raise PR to dev branch first.
+3. Always squash merge the PR you raise except in case of PR to master branch
```

Comment:
Can do squash merge with proper comit message

Reply:
@abhilash-aot rephrasing as:

`Always squash merge the PR you raise with proper commit message except in case of PR to master branch`

Is it fine?

Reply:
Yup looks good. Need all feature specific commits under one. And all commit message should be self explanatory to the code under it.

Reply:
Resolved concerns.